DQM/L1TMonitor/src/L1TRate.cc

```diff
@@ -24,7 +24,7 @@
 #include "CondFormats/DataRecord/interface/L1GtTriggerMenuRcd.h"
 #include "CondFormats/DataRecord/interface/L1GtPrescaleFactorsAlgoTrigRcd.h"
 
-#include "DataFormats/Histograms/interface/MEtoEDMFormat.h"
+//#include "DataFormats/Histograms/interface/MEtoEDMFormat.h"
```

Comment:
Please remove commented out code, or add another line of comment which explains why it must be kept.